### DISCUSSION TITLE
Text Blocks' default text alignment

### DISCUSSION THREAD

**Discussion by RasheedAZ** [JulianPrieber/littlelink-custom#441] [Feature requests]
Hi. Thanks for making this beautiful app. I was wondering if you think it may be a better idea to have the text inside the Text Blocks aligned to the left (or right for RTL languages), instead of center. 

Right now, everything, including lists are center aligned which makes lists look akward with the buttons on the far left and the text in the middle :)

**Comment by JulianPrieber** [comment]
This can be changed with themes. The default will remain centered.
In previous versions you where able to change the text alignment but that was removed with version 3.0 when we switched to a new text editor for text blocks.
